src/main/java/com/spotinst/sdkjava/model/bl/elastigroup/aws/ImportASG.java

```diff
@@ -0,0 +1,156 @@
+package com.spotinst.sdkjava.model.bl.elastigroup.aws;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * @author: Saikumar Jalda
+ * @since: 14/02/2022
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ImportASG implements IPartialUpdateEntity {
```

Comment:
This class is not required to use the interface IPartialUpdateEntity .. Please remove